OpenTrace/Main/View Controllers/HomeViewController.swift

```diff
@@ -76,58 +73,23 @@ class HomeViewController: UIViewController {
             self.pushNotificationGranted = pnsGranted
 
             self.allPermissionOn = self.blePoweredOn && self.bleAuthorized && self.pushNotificationGranted
-
-            self.togglePermissionViews()
         }
     }
 
-    private func toggleIncompleteHeaderView() {
-        successHeaderView.isVisible = self.allPermissionOn
-        powerSaverCardView.isVisible = self.allPermissionOn
-        incompleteHeaderView.isVisible = !self.allPermissionOn
-        appPermissionsLabel.isVisible = !self.allPermissionOn
-    }
+	@IBAction func TrackMyConditionButtonTapped() {
```

Comment:
shouldn't have capital T. but don't worry about it. i'll resolve conflicts and fix